### DISCUSSION TITLE
how to put extension

### DISCUSSION THREAD

**Discussion by rafafranco2203** [seleniumbase/SeleniumBase#2620] [General]
Even after entering the correct path, the extension is not appearing in the browser

from seleniumbase import Driver

local = 'Cookie-Editor.zip'
driver = Driver(browser='chrome', extension_zip=local)

**Comment by mdmintz** [comment]
Your extension might not be properly formatted.
I tried with all the extensions in [seleniumbase/extensions](https://github.com/seleniumbase/SeleniumBase/tree/master/seleniumbase/extensions) and all were loaded successfully.
